pandas/core/tools/datetimes.py

```diff
@@ -437,6 +437,32 @@ def _array_strptime_with_fallback(
 ) -> Optional[Index]:
     """
     Call array_strptime, with fallback behavior depending on 'errors'.
+
+    Used to call array_strptime with the ability for the user to specify
+    error controls and methods for how to handle the thrown errors. 
+    Uses default values to allow the user to have the option to specify
+    format and exactness.
+
+    Parameters
+    ----------
+    arg : list, tuple, ndarray, Series, Index
+        Date to be parsed.
+    name : object
+        None or string for the Index name.
+    tz : timezone value
+        Timezone string value or None.
+    fmt : string or None
+        Holds format value for the datetime arguement.
```

Comment:
I would try to avoid mixing verbal style, e.g:

```
arg : bool : indicates whether to include factor [good]
arg : bool : holds a value which indicates whether to include factor [bad]
arg : bool : boolean for arg [bad] 
arg : bool : is an argument which determines if factor is used [bad]
arg : bool : determines if factor is used [good]
```
```suggestion
    fmt : str, optional
        Datetime format of arg
```